man/man8/zpool.8

```diff
@@ -1542,14 +1542,33 @@ base 1024.  To get the raw values, use the \fB-p\fR flag.
 .sp
 .ne 2
 .na
-\fB\fB-c\fR \fBCMD\fR
+\fB\fB-c\fR \fB[SCRIPT1,SCRIPT2,...]\fR
 .ad
 .RS 12n
-Run a command on each vdev and include first line of output
+Run a script (or scripts) on each vdev and include the output in zpool iostat
 .sp
-The \fB-c\fR option allows you to run an arbitrary command on each vdev and
-display the first line of output in zpool iostat.  The following environment
-vars are set before running each command:
+The \fB-c\fR option allows you to run script(s) each vdev and display the
+output in zpool iostat.  For security reasons, only scripts in the
+/<etc>/zfs/zpool.d directory can be run, and only as an unprivileged user.
+However, a privilaged user can run with \fB-c\fR if they have the
```

Comment:
Privileged is misspelled. Though, the line may sound better as `However, a user can run \fB-c\fR with elevated privileges if they have the ZPOOL_SCRIPTS_AS_ROOT environment variable set.` Thoughts?